ValveResourceFormat/Resource/ResourceTypes/ModelAnimation/MotionFlag.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ValveResourceFormat.ResourceTypes.ModelAnimation
+{
+    [Flags]
+    public enum MotionFlag
```

Comment:
```suggestion
    public enum EMotionFlag
```

And how did you come with up these value names? I see there's missing values.

Reply:
These were the values that I've managed to reproduce using ModelDoc through the ExtractMotion node. The missing values between TZ and RZ are most likely RX and RY, but I didn't want to add guesses.

Reply:
This would be the first prefixed enum name in the solution this way, should I go ahead with renaming it?

Reply:
Whoops, I had a different project in mind.

Let's put it into Resource/Enums as `ModelAnimationMotionFlags` instead.